eng/pipelines/templates/jobs/archetype-sdk-mgmt.yml

```diff
@@ -32,6 +40,9 @@ jobs:
     variables:
       DotNetCoreSDKVersion: '2.2.202'
       skipComponentGovernanceDetection: true
+      MsBuildLogDir: msbuildlogs
+      LoggingArgs: '/v:m /clp:ShowtimeStamp /flp:LogFile=$(MsBuildLogDir)/msbuild.normal.log;Verbosity=normal /flp1:Summary;Verbosity=minimal;LogFile=$(MsBuildLogDir)/msbuild.sum.log /flp2:warningsonly;logfile=$(MsBuildLogDir)/msbuild.wrn.log /flp3:errorsonly;logfile=$(MsBuildLogDir)/msbuild.err.log'
```

Comment:
any reason we need multiple variables for loging arguments.
Can we not reuse the same variable

Reply:
@shahabhijeet , I could not find a way to define shared variables in this jobs template yml, so I move shared variables to top level mgmt.yml, please let me know if any concern.